debug:
    logging:
        configuration: info
        initialization: info
        collision_space: info
        kinematics: info
        hashmanager: info
        search: info
        motionprimitives: info
        heuristics: info
        pathpostprocessor: info
